packages/lesswrong/components/dialogues/DialogueRecommendationRow.tsx

```diff
@@ -308,7 +322,7 @@ const DialogueRecommendationRow = ({ rowProps, classes, showSuggestedTopics }: D
   });
 
   const { loading: postsLoading, error: postsError, data: postsData } = useQuery(gql`
-    query UsersReadPostsOfTargetUser($userId: String!, $targetUserId: String!, $limit: Int) {
+    query UsersReadPostsOfTargetUser($userId: String!, $targetUserId: String!, $limit: Int!) {
```

Comment:
My guess is that all of these new-ish queries are missing appropriate `skip` logic to ensure they aren't run for logged out users